lib/slack_interface/request_handler.coffee

```diff
@@ -127,6 +127,14 @@ class SlackInterfaceRequestHandler
                     reply_data['text'] = ['Ok.', 'Sweet.', 'Chur.', 'Done like dinner.', 'sorted.org.nz (use your mouse!)', 'Coolies.', 'No problem, brah.', 'Affirmative.', 'Gotcha.', 'Aye-aye, captain! :captain:'][Math.floor(Math.random() * 10)]
                   else
                     reply_data['text'] = "Oops, you did it again. Try `help` if you need some."
+                else if @auth.args[0] == '|' and @auth.args[1] == 'grep'
```

Comment:
I'd remove the need for `grep` - introduces some weird edge cases where you could do `list | notgrep` etc. While I appreciate turning it into a terminal, I think either `list | <searchterm>` or even just `search <searchterm>` or `searchlist <searchterm>` might be better here.

There's some argument for `list search <searchterm>` too - you'd just have to blacklist it as a playlist name.